libcontainer/cgroups/systemd/common.go

```diff
@@ -366,16 +366,24 @@ func systemdVersion(conn *systemdDbus.Conn) (int, error) {
 		}
 
 		// verStr is like "v245.4-1.fc32" (including quotes)
-		if !strings.HasPrefix(verStr, `"v`) {
```

Comment:
The comment above needs to be fixed as well, to mention that new form you discovered.